app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -1,13 +1,25 @@
 .gem-c-big-number {
```

Comment:
I'd suggest including the `govuk-text-colour` mixin too – black text should be `#0b0c0c`, but at the minute this will be using the user agent default text colour. That'll probably be `#000000` in most cases, but may not always be, which could cause accessibility issues – see https://www.w3.org/TR/WCAG20-TECHS/F24.html.